lib/cards/KCardGrid.vue

```diff
@@ -77,6 +136,47 @@
           return [LAYOUT_1_1_1, LAYOUT_1_2_2, LAYOUT_1_2_3].includes(value);
         },
       },
+      // eslint-enable-next-line kolibri/vue-no-unused-properties
+      /**
+       * Overrides the base grid `layout` for chosen breakpoints levels
+       */
+      // eslint-disable-next-line kolibri/vue-no-unused-properties
+      layoutOverride: {
```

Comment:
I am not finding the rationale for doing this as an array rather than an object keyed by breakpoint, as it was in the [commit linked](https://github.com/learningequality/kolibri-design-system/commit/61e95ea084127811a58d880f7ee1a4e0f131c113) in the beta proposal.

My only real concern is that it felt hard to reason about what the resulting layout would be, because of the ability to mix in multiple layouts. Having the object specced form is more verbose, but I think easier to understand exactly what is happening.

Have I missed something else in the motivation here?

Reply:
For now, I at least attempted to improve documentation by making more clear the bottom-up strategy

Reply:
For now, I at least attempted to improve documentation by making more clear the bottom-up strategy is intended